src/components/accounts/two_factor/EnableTwoFactor.js

```diff
@@ -190,7 +204,14 @@ export function EnableTwoFactor(props) {
                     >
                         <Translate id={`button.continue`} />
                     </FormButton>
-                    <FormButton className='link' type='button' linkTo='/profile'><Translate id='button.skip' /></FormButton>
+                    <FormButton 
+                        className='link' 
+                        type='button' 
+                        linkTo='/profile' 
+                        onClick={() => Mixpanel.track("2FA Click Skip button", {url_link:"/profile"})}
```

Comment:
nit: I am a bit confused around event naming. Sometimes words are capitalised, sometimes they don't. Is there some pattern that needs to be followed?

Reply:
I will follow the pattern for verb to be capitalized and other not